src/main.cpp

```diff
@@ -91,17 +86,17 @@ bool parse_config(char * config_file, server_options *server_options_pointer) {
 
   get_server_options(config, server_options_pointer);
   std::string port = server_options_pointer->port;
-  std::string static_file_root = server_options_pointer->static_file_root;
+  //std::string static_file_root = server_options_pointer->static_file_root;
 
   if (port == "") {
     std::cerr << "Please specify a port.\n";
     return false;
   }
```

Comment:
The way I interpreted the spec was there was only one static file handler and it could have multiple different paths/roots, e.g. "You should support multiple URL paths for a request handler". But I'm not entirely sure, it's vague